bundles/GraphicServiceOrder/Form/GraphicServiceOrderForm.php

```diff
@@ -81,34 +82,30 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'allow_delete' => true,
                 'entry_options' => ['label' => false],
             ])
-            ->add('description', TextareaType::class, [
-                'label' => 'service_order_form.job_description.description.label',
-                'attr' => ['class' => 'form-control', 'required'],
-                'help_attr' => ['class' => 'form-text text-muted'],
-                'help' => 'service_order_form.job_description.description.help',
-                'required' => false,
-            ])
-      // Using OneupUploaderBundle and ajax for uploading the files, see GsUploadListener and jquery-fileupload-config.js
-            ->add('files', FileType::class, [
+            ->add('multi_upload', CollectionType::class, [
                 'label' => 'service_order_form.job_description.files.label',
-                'constraints' => [
-                    new All([
+                'entry_type' => FileType::class,
+                'allow_add' => true,
+                'allow_delete' => true,
+                'mapped' => false,
+                'entry_options' => [
+                    'label' => false,
+                    'required' => false,
+                    'constraints' => [
                         new File([
-                            'maxSize' => getenv('FORM_FILE_GS_UPLOAD_SIZE'),
+                            'maxSize' => $_ENV['FORM_FILE_GS_UPLOAD_SIZE'],
```

Comment:
You should (probably) not use `$_ENV` here. This setting should be injected as an argument to the constructor:

```php
    …
    private $params;

    public function __construct(ParameterBagInterface $params)
    {
        $this->params = $params;
    }
    …
```
(cf. https://symfony.com/blog/new-in-symfony-4-1-getting-container-parameters-as-a-service)

Reply:
Fixed